app/uk/gov/hmrc/rasapi/controllers/LookupController.scala

```diff
@@ -149,28 +149,27 @@ class LookupController @Inject()(
                     userId = id)
                   Logger.error(s"[LookupController][getResidencyStatus] Error occurred for userId ($id), " +
```

Comment:
Not changed but from line 100-142, there are a lot of debug logs that should be adjusted to a different level based on what status they throw.